README.md

```diff
@@ -97,6 +97,17 @@ This re-renders all of the currently-displayed elements, updating them from thei
 
 This generally needs to be called any time the data to be displayed changes. This includes additions, removals, and modifications to the data. See our [examples below](#data-manipulation-using-itemschanged) for more information.
 
+### `scrollToIndex(index: number, position: string = "start")` method

```

Comment:
maybe rename `position` to `viewportPosition`? @domenic wdyt?

Reply:
It feels a bit like the `align` properties for flexbox. The position here also depends on the scroll direction.

Maybe you want to change the names of the values instead? `position: "viewport-start"` or `position: "scroll-direction-start"`

Reply:
I see that `scrollIntoView` uses  `"start", "center", "end", or "nearest"` so let's keep that

> alignToTop: If true, the top of the element will be aligned to the top of the visible area of the scrollable ancestor. Corresponds to scrollIntoViewOptions: {block: "start", inline: "nearest"}. This is the default value.

Here the platform is using `block` and `inline` which is more a what than `position`. It seems a bit more future proof to create a `scrollToIndexOptions` dictionary

Reply:
+1 on using an options dictionary. I didn't realize that there was also a "behavior" member (hidden in https://drafts.csswg.org/cssom-view/#dictdef-scrolloptions) which we may want to add in the future.

That still leaves the question of what to call it. I think "position" is pretty good though. "Scroll to index 5 at the end position" sounds OK when said out loud.

(Working on updating this now.)